### ISSUE TITLE
missing i18n of action list (\editor-client\src\js\ui\actionList.js)

### ISSUE THREAD

**Issue by heikokue** [node-red/node-red#2808]
The Action List is currently build in a generic way by JS in actionList.js, line 171+.
So it's not possible to do the i18n for the Action List.
The actionList.js is also missing in the design note: https://github.com/node-red/node-red/wiki/Design%3A-i18n
The generation of the Action List shall use a locales json table.

- [x] Node-RED version: V1.2.6
- [ ] Node.js version:
- [ ] npm version:
- [ ] Platform/OS:
- [ ] Browser:

**Comment by knolleary** [comment]
Is this something you are working on fixing? If so, please discuss a design with me before you get too far into it. Want to make sure your proposed approach is suitable.

**Comment by heikokue** [comment]
No, I just wanted to report that the Action List was forgotten for the i18n expansion. Due to a lack of JS knowledge, I cannot change that myself.
